account_ecotax/models/account_ecotax_classification.py

```diff
@@ -0,0 +1,92 @@
+# © 2014-2023 Akretion (http://www.akretion.com)
+#   @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountEcotaxClassification(models.Model):
+    _name = "account.ecotax.classification"
+    _description = "Account Ecotax Classification"
+
+    name = fields.Char(required=True)
+    code = fields.Char()
+    ecotax_type = fields.Selection(
+        [("fixed", "Fixed"), ("weight_based", "Weight based")],
+        required=True,
+        help="If ecotax is weight based,"
+        "the ecotax coef must take into account\n"
+        "the weight unit of measure (kg by default)",
+    )
+    ecotax_coef = fields.Float(
+        digits="Ecotax", compute="_compute_ecotax_vals", readonly=False, store=True
+    )
+    default_fixed_ecotax = fields.Float(
+        digits="Ecotax",
+        help="Default fixed ecotax amount.",
+        compute="_compute_ecotax_vals",
+        readonly=False,
+        store=True,
+    )
+    categ_id = fields.Many2one(
+        comodel_name="account.ecotax.category",
+        string="Category",
+    )
+    sector_id = fields.Many2one(
+        comodel_name="ecotax.sector",
+        string="Ecotax sector",
+    )
+    collector_id = fields.Many2one(
+        comodel_name="ecotax.collector",
+        string="Ecotax collector",
+    )
+    active = fields.Boolean(default=True)
+    company_id = fields.Many2one(
+        comodel_name="res.company",
+        default=lambda self: self.env.company,
+        help="Specify a company"
+        " if you want to define this Ecotax Classification only for specific"
+        " company. Otherwise, this Fiscal Classification will be available"
+        " for all companies.",
+    )
+    product_status = fields.Selection(
+        [("M", "Menager"), ("P", "Professionnel")],
```

Comment:
Would it make sense to use English here combined with `fr.po` ?
```suggestion
        [("M", "Domestic"), ("P", "Professional")],
```